frontend/src/composables/useNormHtml.ts

```diff
@@ -28,3 +36,34 @@ export function useNormHtml(
 
   return readonly(normHtml)
 }
+
+export function useNormHtmlByEli(
+  eli: MaybeRefOrGetter<string | undefined>,
+  at?: MaybeRefOrGetter<Date | undefined>,
+): {
+  normHtml: Readonly<Ref<string | undefined>>
+  error: Readonly<Ref<Error | null>>
+  isFetching: Readonly<Ref<boolean>>
+} {
+  const eliRef = ref(toValue(eli))
+  const atRef = ref(toValue(at))
+
+  watch(
+    () => [toValue(eli), toValue(at)],
+    () => {
+      eliRef.value = toValue(eli)
+      atRef.value = toValue(at)
+    },
+    { immediate: true },
+  )
```

Comment:
we should be able to remove this watcher and the two refs and be able to just use `eli` and `at` immediately.